### PR TITLE
chore: add collab checkboxes to issue templates

### PR DESCRIPTION
**Pull request by bastimeyer**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/3_plugin_request.yml [hunk 1]
```diff
@@ -33,32 +33,51 @@ body:
           required: true
         - label: "[I have checked the list of open and recently closed plugin requests](https://github.com/streamlink/streamlink/issues?q=is%3Aissue+label%3A%22plugin+request%22)"
           required: true
+  - type: checkboxes
+    attributes:
+      label: Collaboration
+      options:
+        - label: My request is genuine, in the best interest of the project, and I have access to the stream(s) in my web browser
```
Comment (gravyboat) [inline .github/ISSUE_TEMPLATE/3_plugin_request.yml:40]: Does this need to be expanded to "without the usage of a VPN"?

**Review by gravyboat — APPROVED** [review]
Changes look good. One discussion point about VPN usage since it seems like we've been getting a lot more sites recently that only work with VPNs but up to you. 👍

File: .github/ISSUE_TEMPLATE/3_plugin_request.yml (hunk 1 above)
Comment (bastimeyer) [reply .github/ISSUE_TEMPLATE/3_plugin_request.yml:40]: I was thinking about it, but there's the access restriction question down below

Comment (gravyboat) [reply .github/ISSUE_TEMPLATE/3_plugin_request.yml:40]: Works for me! Can always update it if needed.

**Pull request merged**